WordPress/Classes/ViewRelated/Reader/ReaderSaveForLaterAction.swift

```diff
@@ -32,11 +32,15 @@ final class ReaderSaveForLaterAction {
         let readerPostService = ReaderPostService(managedObjectContext: context)
 
         readerPostService.toggleSavedForLater(for: post, success: {
-            self.presentSuccessNotice(for: post, context: context, origin: origin, completion: completion)
+            if origin == .otherStream {
+                self.presentSuccessNotice(for: post, context: context, origin: origin, completion: completion)
+            }
             completion?()
-            }, failure: { error in
+        }, failure: { error in
+            if origin == .otherStream {
                 self.presentErrorNotice(error, activating: !post.isSavedForLater)
-                completion?()
+            }
+            completion?()
```

Comment:
I don't think we should show the success/failure notice if a user unsaves a post from the `Saved` tab for the following reasons:

- The `ReaderSavedPostUndoCell` already indicates to the user that the post has been "unsaved"
- The notice appears when a user navigates _away_ from the `Saved` tab... so the notice seems out of context

Let me know what you think! 😄

Reply:
Sounds reasonable. I think to the `ReaderSavedPostUndoCell` is sufficient.